internal/rpc/rpc.go

```diff
@@ -268,3 +268,12 @@ func RetryStreamingServerStream[Req, Resp any](
 
 	}
 }
+
+// useDebugErrorLevel indicates whether the specified error should be reported as a debug
+// level log.
+func logLevelForError(err error) log.Level {
+	if err != nil && strings.Contains(err.Error(), "connect: connection refused") {
```

Comment:
It's better to use `errors.Is` for this check, like: https://go.dev/play/p/M0X4W1wcSQn

Sorry I missed this earlier!